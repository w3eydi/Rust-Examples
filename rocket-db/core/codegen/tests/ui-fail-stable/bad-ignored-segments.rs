../ui-fail/bad-ignored-segments.rs