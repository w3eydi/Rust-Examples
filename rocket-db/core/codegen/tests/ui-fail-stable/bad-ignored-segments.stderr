error: parameter must be named
  --- help: use a name such as `_guard` or `_param`
 --> $DIR/bad-ignored-segments.rs:6:7
  |
6 | #[get("/c?<_>")]
  |       ^^^^^^^^

error: parameter must be named
  --- help: use a name such as `_guard` or `_param`
 --> $DIR/bad-ignored-segments.rs:9:21
  |
9 | #[post("/d", data = "<_>")]
  |                     ^^^^^
